pros/conductor/conductor.py

```diff
@@ -279,13 +288,15 @@ def remove_template(project: Project, identifier: Union[str, BaseTemplate], remo
                                     remove_empty_directories=remove_empty_directories)
 
     def new_project(self, path: str, no_default_libs: bool = False, **kwargs) -> Project:
-        self.is_beta = kwargs.get('beta', False)
+        self.use_pros4 = kwargs.get('modern', False)
+        if not self.use_pros4 and self.warned_pros3_deprecated:
+            ui.echo(f"PROS 3 is deprecated. PROS 4 is now the latest version. "
+                    f"To use PROS 4 use the --modern flag.")
+
         if Path(path).exists() and Path(path).samefile(os.path.expanduser('~')):
             raise dont_send(ValueError('Will not create a project in user home directory'))
-        for char in str(Path(path)):
-            if char in ['?', '<', '>', '*', '|', '^', '#', '%', '&', '$', '+', '!', '`', '\'', '=',
-                        '@', '\'', '{', '}', '[', ']', '(', ')', '~'] or ord(char) > 127:
-                raise dont_send(ValueError(f'Invalid character found in directory name: \'{char}\''))
+        if re.match(r'^[\w\-. /]+$', str(Path(path))) is None:
+            raise dont_send(ValueError('Invalid characters found in path'))
```

Comment:
nice improvement, lets put this in another PR we can track the changes in the next update